python/cudf/pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "pandas>=1.3,<1.6.0dev0",
     "protobuf>=4.21,<5",
     "ptxcompiler",
-    "pyarrow==12.0.0.*",
+    "pyarrow==12.*",
```

Comment:
Why do we specify `12.0.1.*` above and here we're ok with `12.*`?

Reply:
Nevermind, I see https://github.com/rapidsai/cudf/pull/14319/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6R264-R265 explains that.